### PR TITLE
fix(scripts): 'build' script doesn't require GITHUB_ACTION's env. variables to run in local

### LINKED ISSUE #185: Running `npm run build` in local does not work out of the box
**What's wrong?**
Running `npm run build` in local environment does not work out of the box. It requires `$ANICCA_REPOSITORY_PATH` to be set

**What did you expect?**
To be able to run `scripts/build.js` without the need of defining environment variables related to GitHub Actions. Also, a better description of `anicca` usage would be awesome so it is understood how to use  `$ANICCA_REPOSITORY_PATH`.

### PR DESCRIPTION
**Pull request by oscard0m**
# Description
When running `npm run build` in local environment:

- Requires to setup `$GITHUB_WORKSPACE` only when it's GitHub Actions environment
- Requires to setup `$ANICCA_REPOSITORY_PATH` only when it's local environment
 
# Context
When running `npm run build` in local environment is not possible unless you define `$GITHUB_WORKSPACE` and `$ANICCA_REPOSITORY_PATH`.

Fixes #185 #186

### CODE REVIEW

**Comment by gr2m** [comment]
Thanks for your help!

>     * Fallbacks to _current directory_ if `$ANICCA_REPOSITORY_PATH` is not defined (@gr2m not sure if this is correct or this variable should always have a certain path, if so, let me know and I will improve the `CONTRIBUTING.md` to explain that.

Only `ANICCA_REPOSITORY_PATH` is required when running locally. It has to be set to the path of a local clone of https://github.com/xuorig/anicca

File: scripts/build.js [hunk 1]
```diff
@@ -11,12 +11,12 @@ const mapObj = require("map-obj");
...
-if (!process.env.ANICCA_REPOSITORY_PATH && !process.env.GITHUB_WORKSPACE) {
-  throw new Error("Please set ANICCA_REPOSITORY_PATH or GITHUB_WORKSPACE");
-}
-
 run();
 
+if (!process.env.GITHUB_ACTIONS && !process.env.ANICCA_REPOSITORY_PATH) {
+  throw new Error("Please set ANICCA_REPOSITORY_PATH");
+}
+
```
Comment (gr2m) [inline scripts/build.js:19]: can you revert this change? We have to check for the env variables before calling `run()`

**Review by gr2m — APPROVED** [review]
Thanks 👍🏼

File: scripts/build.js (hunk 1 above)
Comment (oscard0m) [reply scripts/build.js]: Moved lines before `run()`

—-

Be aware I modified the check considering it’s not necessary to check the existance of GITHUB_WORKSPACE env var. When my new code resolves aniccaCwd will check if it’s GitHub Action or not and if it is, that env var will exist by definition. This way we don’t enforce users to have that env var defined in local.

**Pull request merged**